Patches/Vanilla Factions Expanded - Pirates/ThingDefs_Misc/ThingDefs_Artillery.xml

```diff
@@ -117,6 +124,13 @@
 
       <!-- Cannon - Based on the M1897 12-pounder -->
 
+      <li Class="PatchOperationReplace">
+        <xpath>/Defs/ThingDef[@Name = "VFEP_BaseCannonBuilding"]/fillPercent</xpath>
+        <value>
+            <fillPercent>0.90</fillPercent>
```

Comment:
0.65 was done on purpose in my PR. cannons are not that tall IRL and them being tall enough to only be able to be shot behind sandbags and not embrasures was the whole point